en/02_Developer_Guides/01_Templates/09_Casting.md

```diff
@@ -150,15 +146,12 @@ Within the template, fields can have their encoding customised at a certain leve
 See the [formatting](#formatting) section above for some of the more common formatting methods available
 and how they affect escaping.
 
-[hint]
-If you are unsure of whether the field has been cast to `HTMLText` but you know
-it contains safe HTML content, you can use `.RAW` to ensure the HTML is not escaped.
-[/hint]
-
-[warning]
-Be careful using `.RAW` on non HTML field types - if the value being formatted includes content provided
-by the user you could be introducing attack vectors for cross-site scripting attacks.
-[/warning]
+> [!TIP]
+> If you are unsure of whether the field has been cast to `HTMLText` but you know
+> it contains safe HTML content, you can use `.RAW` to ensure the HTML is not escaped.
+>
+> But be careful using `.RAW` on non HTML field types - if the value being formatted includes content provided
+> by the user you could be introducing attack vectors for cross-site scripting attacks.
```

Comment:
Combined into a single tip since they're talking about the same thing, and to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.